websites/docs-smoke-test/src/content/code-samples/code-block.mdx

```diff
@@ -9,6 +9,15 @@ The following block-level code samples must be
 - have a copy button
 - do not have a header bar because they don't have a title and are single-language
 
+### Terminal/Console
+
+```console{outputLines: 2-4}
```

Comment:
This is how it works, by default all lines have the "prompt" line and you can "hide it" by providing a line or range of line.

In this case only the first line has the prompt.

Reply:
We should also use `console`, as otherwise `bash`, `shell` etc should be used for scripting.